src/libs/ajax/teaspoons/teaspoons-models.ts

```diff
@@ -15,3 +15,20 @@ export interface PipelineQuotaWithDetails {
   quotaConsumed: number;
   quotaUnits: string;
 }
+
+export interface PipelineRun {
+  jobId: string;
+  pipelineName: string;
+  status: PipelineRunStatus;
+  description?: string;
+  timeSubmitted: string;
+  timeCompleted?: string;
+}
+
+export interface GetPipelineRunsResponse {
+  totalResults: number;
```

Comment:
`totalResults` is not currently provided by the Teaspoons API, but I have a branch in that repo to add it